README.md

```diff
@@ -220,12 +216,17 @@ To understand what we're trying to become, it may be useful to understand what
 we're trying to avoid. See effects of [contempt](http://blog.aurynn.com/86/contempt-culture)
 and [competition](https://medium.com/tech-diversity-files/if-you-think-women-in-tech-is-just-a-pipeline-problem-you-haven-t-been-paying-attention-cb7a2073b996#.yjrp4mqkb) on environments.
 
-#### Favour experimentation over argument
+#### Favour experimentation over opinion
```

Comment:
Fair point, I kinda lost focus when changing this expression. My thinking was around that some arguments based on facts and could have a place, like classic "develop mobile first since in many cases users access from mobile is already more than desktop". Still, i don't think this suggestion can convey much of this idea so dropping it makes sense :dancers: